std_msgs/Bool demo
---
rob9/GraspMsg[] grasps
